make/autoconf/flags-ldflags.m4

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2011, 2018, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please don't change the Oracle copyright notice. Instead, add an IBM copyright notice (like appears in later versions):
```

# ===========================================================================
# (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
# ===========================================================================
```